utilities/host-setup-tasks.sh

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env bash
 
+[ -z "${application_ready}" ] && { echo "[FATAL] You need to source 'utilities/application.sh' before sourcing this script."; return 1; }
```

Comment:
Is this addition intentional?

Reply:
Yes, as (1) it reflects usage as far as I could find, (2) `_log_` is defined in `application.sh`, so it would fail anyway without it.

Reply:
Good.